opt_out_http_api/tests/test_api.py

```diff
@@ -18,7 +19,7 @@ def tearDown(self):
 
     @inlineCallbacks
     def start_server(self):
-        self.app = API()
+        self.app = API(backend_class=OptOutMemory)
```

Comment:
If you pass in an instance instead of a class into your app, you can nicely clean up some of the tests:
```
self.store = OptOutMemory()
self.app = API(self.store)  # <-- this requires changing API.__init__
```
Then when you need to add an opt out you can do:
```
opt_out = self.store.put("twitter_handle", "@example")
```
Or when you need to check that something made it into the opt out store:
```
opt_out = self.store.get(...)
```